lib/install/config/webpacker.yml

```diff
@@ -35,7 +36,13 @@ test:
 
   public_output_path: packs-test
 
+  compile_missing_packs: true
+
+  compiler_cache_path: tmp/.webpacker-checksum
+  compiler_tracked_paths:
+    - app/javascript/**/*
+    - yarn.lock
+    - package.json
```

Comment:
Perhaps make these the defaults internally and add an API for watching additional paths. Like Spring does: https://github.com/rails/spring#watching-files-and-directories